src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -2487,7 +2608,7 @@ public void run()
     /**
      * Drops current memtable without flushing to disk. This should only be called when truncating a column family which is not durable.
```

Comment:
nit: Might be worth mentioning in the comment that this *can* be called if the table has durable writes but the `Memtable` is also durable?

Reply:
Added.